core/lib/refinery/core/configuration.rb

```diff
@@ -32,6 +32,7 @@ module Core
     self.visual_editor_javascripts = []
     self.visual_editor_stylesheets = []
     self.plugin_priority = []
+    self.regex_white_list = false
```

Comment:
All configs for SectionPresenter are in the `pages/lib/refinery/pages.configuration`. I'm in favor to move `regex_white_list ` in Pages and rename it `whitelist_regex` to match current configs.

Could you add by default your regex?